src/common/SettingsAPI/settings_helpers.cpp

```diff
@@ -77,4 +78,34 @@ namespace PTSettingsHelper
         result = result.append(log_settings_filename);
         return result.wstring();
     }
+
+    bool get_oobe_opened_state()
```

Comment:
Let's make sure that this function doesn't throw exceptions, use the `noexcept` operator and if needed, catch possible exceptions.
Same for `save_oobe_opened_state()`